demisto_sdk/commands/lint/linter.py

```diff
@@ -654,7 +655,7 @@ def _docker_image_create(self, docker_base_image: List[Any]) -> Tuple[str, str]:
         # Trying to pull image based on dockerfile hash, will check if something changed
         errors = ""
         identifier = hashlib.md5("\n".join(sorted(pip_requirements)).encode("utf-8")).hexdigest()
-        test_image_name = f'devtest{docker_base_image[0]}-{identifier}'
+        test_image_name = f'{docker_base_image[0].replace("demisto", "devtestdemisto")}-{identifier}'
```

Comment:
why?

Reply:
The base image name is `{docker-art...}/demisto/image`, we want to use `{docker-art...}/devtestdemisto/image`